partner_communication/wizards/change_text_wizard.py

```diff
@@ -29,11 +28,12 @@ def _default_text(self):
         lang = list(set(communications.mapped('partner_id.lang')))
         if len(config) != 1:
             raise UserError(
-                _("You can only update text on one communication type at "
+                ("You can only update text on one communication type at "
```

Comment:
No! Put back that `_` and put it back from the imports. This is a translation function that translates the error strings to the user language.